### ISSUE TITLE
remix grunge stack aws

### ISSUE THREAD

**Issue by riadraz** [remix-run/grunge-stack#179]
### Have you experienced this bug with the latest version of the template?

npx create-remix@latest --template remix-run/grunge-stack

### Steps to Reproduce

HP-ProBook-450-G3:~$ npx create-remix@latest --template remix-run/grunge-stack
(node:24084) [DEP0040] DeprecationWarning: The `punycode` module is deprecated. Please use a userland alternative instead.
(Use `node --trace-deprecation ...` to show where the warning was created)

 remix   v2.4.0 💿 Let's build a better website...

   dir   Where should we create your new project?
         ./remixawstest

### Expected Behavior

Setup finishes without errors. Go ahead..

### Actual Behavior

      ◼  Template: Using remix-run/grunge-stack...
      ✔  Template copied

   git   Initialize a new git repository?
         Yes

  deps   Install dependencies with npm?
         Yes

  init   This template has a remix.init script. Do you want to run it?
         Yes

 ██████  Dependencies installing with npm...

      ▲  Oh no! Failed to install dependencies.

**Comment by cypherpower** [comment]
I have a pull request submitted that will fix this but meanwhile you can update your package.json to use vite: ^5.0.0 instead of ^4.5.0

**Comment by riadraz** [comment]
yes i edited the vite verion to 5.0.0 before running npm install and it worked fine with all test passed. This have fixed all github action workflow error and failed to deployment reason too. I am getting another one after deploying in aws that is ... INTERNAL SERVER ERROR ..

**Comment by riadraz** [comment]
> I have a pull request submitted that will fix this but meanwhile you can update your package.json to use vite: ^5.0.0 instead of ^4.5.0

if any help about internal server error after deploying in aws, on hitting the url got from deployment work flow.

**Comment by riadraz** [comment]
I evetually deployed grunge stack in aws. I constantly got  "Internal server error" . 

The steps are:

Before running the : 
npx arc env --add --env staging ARC_APP_SECRET $(openssl rand -hex 32)
  npx arc env --add --env staging SESSION_SECRET $(openssl rand -hex 32)
  npx arc env --add --env production ARC_APP_SECRET $(openssl rand -hex 32)
  npx arc env --add --env production SESSION_SECRET $(openssl rand -hex 32)

Did this:
make sure your secret variaabls (AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY) are same both in your CLI ["aws configure" command will ask for those keys and you have to get it from aws IAM user first.. ] , github and aws then run the 4 line command and commit it to github. Deploy.yml will work and make cloud formation. I got the link in deploy workflow https://t1hu92wtqh.execute-api.us-west-2.amazonaws.com/
